utilities/storage.py

```diff
@@ -569,15 +569,15 @@ def data_volume_template_dict(
     return dv.res
 
 
-def data_volume_template_with_source_ref_dict(data_source, storage_class=None):
+def data_volume_template_with_source_ref_dict(data_source, storage_class=...):
     source_dict = data_source.source.instance.to_dict()
     source_spec_dict = source_dict["spec"]
     dv = DataVolume(
         name=data_source.name,
         namespace=data_source.namespace,
         size=source_spec_dict.get("resources", {}).get("requests", {}).get("storage")
         or source_dict.get("status", {}).get("restoreSize"),
-        storage_class=storage_class or source_spec_dict.get("storageClassName"),
+        storage_class=storage_class if storage_class is not ... else source_spec_dict.get("storageClassName"),
```

Comment:
Why don't we keep `storage_class=None` by default, and do here `storage_class=storage_class`?